test/nodes/test_file_converter.py

```diff
@@ -184,6 +184,24 @@ def test_pdf_parallel_sort_by_position(Converter):
     assert pages[-1] == "This is the page 50 of the document."
 
 
+@pytest.mark.unit
```

Comment:
I think it's not correct marking this test as `unit` given it requires all those dependencies installed in the `tests.yml` workflow.

To make it a real unit test we should mock some stuff and avoid installing all dependencies since we won't run them at all. 

I think this [`test_summarizer.py`](https://github.com/deepset-ai/haystack/blob/main/test/nodes/test_summarizer.py) is a good example of a well structured unit test. It also has a related [`e2e/test_summarizer`](https://github.com/deepset-ai/haystack/blob/main/e2e/nodes/test_summarizer.py). They've been added in #4299.